protoc-gen-swagger/genswagger/template.go

```diff
@@ -653,21 +653,26 @@ func renderServices(services []*descriptor.Service, paths swaggerPathsObject, re
 					}
 					if opts.Security != nil {
 						newSecurity := []swaggerSecurityRequirementObject{}
-						if operationObject.Security == nil {
-							newSecurity = []swaggerSecurityRequirementObject{}
-						} else {
-							newSecurity = operationObject.Security
+						if operationObject.Security != nil {
+							newSecurity = *operationObject.Security
 						}
```

Comment:
Can we not just set `newSecurity = nil` when `operationObject.Security == nil`?

Reply:
I think we really need empty slice in this case.

Reply:
OK